docs/content/en/docs/tasks/troubleshoot/troubleshooting.md

```diff
@@ -534,3 +535,170 @@ $ kubectl get deployments -n eksa-system
+
+1. Change the device list in your cluster yaml configuration file and use the eksctl anywhere upgrade cluster command.
+
+    ```
+    $ eksctl anywhere upgrade cluster -f eks-a-cluster.yaml
+    ```
+
+### Node outage
+
+**Unintentional instance outage**
+
+When an instance is in exception status (for example, terminated/stopped for some reason), it will be discovered automatically by Amazon EKS Anywhere and there will be a new replacement instance node created after 5 minutes. The new node will be provisioned to devices based on even provision strategy. In this case, the new node will be provisioned to a device with the fewest number of machines of the same type. Sometimes, more than one device will have the same number of machines of this type. Thus, we cannot guarantee it will be provisioned on the original device.
+
+**Intentional node replacement**
+
+If you want to replace an unhealthy node which didn't get detected by Amazon EKS Anywhere automatically, you can follow the these steps.
+
+>**_NOTE:_** do not delete all worker machine nodes or control plane nodes or etcd nodes at the same time. Make sure you delete machine nodes one by one.
+
+1. Cordon nodes so no further workloads are scheduled to run on it.
+
+    ```sh
+    $ kubectl cordon node-name --ignore-daemonsets --kubeconfig=cluster-name/cluster-name-eks-a-cluster.kubeconfig              
+    ```
+
+1. Drain machine nodes of all current workloads.
+
+    ```sh
+    $ kubectl drain node-name --ignore-daemonsets --kubeconfig=cluster-name/cluster-name-eks-a-cluster.kubeconfig              
+    ```
+
+1. Delete machine nodes.
+
+    ```sh
+    $ kubectl delete node node-name --kubeconfig=cluster-name/cluster-name-eks-a-cluster.kubeconfig              
+    ```
+
+1. New nodes will be provisioned automatically. You can check the provision result with the get machines command.
+
+    ```sh
+    $ kubectl get machines -A --kubeconfig=cluster-name/cluster-name-eks-a-cluster.kubeconfig              
+    ```
+
+### Cluster Deletion Fails
+
+If your Amazon EKS Anywhere cluster creation failed and the `eksctl anywhere delete cluster -f eksa-cluster.yaml` command cannot be run successfully, manually delete a few resources before trying the command again. Run the following commands from the computer on which you set up the AWS configuration and have the Snowball Edge Client installed. If you are using multiple Snowball Edge devices, run these commands on each.
+
+```sh
+// get the list of instance ids that are created for Amazon EKS Anywhere cluster, 
+// that can be identified by cluster name in the tag of the output
+$ aws ec2 describe-instances --endpoint http://snowball-ip:8008 --profile profile-name
+
+// the next two commands are for deleting DNI, this needs to be done before deleting instance
+$ PATH_TO_Snowball_Edge_CLIENT/bin/snowballEdge describe-direct-network-interfaces --endpoint https://snowball-ip --manifest-file path-to-manifest-file --unlock-code unlock-code
```

Comment:
I'm just curious what we have this "PATH_TO_Snowball_Edge_CLIENT" here. Shouldn't it just be put in the user's $PATH, so they could just type "snowballEdge"?

Reply:
added https://github.com/aws/eks-anywhere/pull/4776/files#diff-d6526994e6cc4a190722462afeac372b842fb107df0e61cedd96788b1878ee3fR552 and https://github.com/aws/eks-anywhere/pull/4776/files#diff-d6526994e6cc4a190722462afeac372b842fb107df0e61cedd96788b1878ee3fR686 for instruction on installing snowball edge client